module/platform/OpenCR/HardwareIO/data/config/HardwareIO.yaml

```diff
@@ -94,3 +94,9 @@ servos:
     direction: 1
     offset: 0.0
     simulated: false
+
+servo:
+  temp_tol: 50.0
+
+buzzer:
+  freq: 880
```

Comment:
frequency, and this is the frequency for the servo temperature alarm so put it in the config with that value